test/Microsoft.Framework.Configuration.Ini.Test/IniConfigurationSourceTest.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Framework.Configuration.Ini.Test
```

Comment:
Rename this file to `IniConfigurationTest.cs` like `EnvironmentVariablesTest.cs` was renamed.